xml/snmp_sysdescr.xml

```diff
@@ -5452,16 +5452,18 @@ Copyright (c) 1995-2005 by Cisco Systems
   <!--======================================================================
                               SonicWall
    =======================================================================-->
-  <fingerprint pattern="^SonicWALL (.*?)\s+\(SonicOS \S+ (\d[^\)]+)\)\s*$">
+  <fingerprint pattern="^SonicWALL (\S+)\s+(\d+).*?\(SonicOS \S+ (\d[^\)]+)\)\s*$">
     <description>SonicWall - SonicOS Enhanced variant</description>
     <example>SonicWALL NSA 220 (SonicOS Enhanced 5.8.1.2-20o)</example>
     <example>SonicWALL TZ 215 wireless-N (SonicOS Enhanced 5.8.1.2-6o)</example>
     <example>SonicWALL TZ190 Enhanced (SonicOS Enhanced 3.6.0.4-30e)</example>
+    <example>SonicWALL SuperMassive 9200 (SonicOS Enhanced 6.2.5.3-35n)</example>
```

Comment:
You should update these examples to ensure they are pulling out the correct groups. 
See an example of how to do that with the attributes here: https://github.com/rapid7/recog/pull/239/files#diff-fbf85d38d785f6c2a3362b7fc1aba5d3R659
You can then check they all match with `rake tests`.

Reply:
Definitely ++ to this

Reply:
:+1: group verification added, rake tests pass